dbml-homepage/docs/js-module/connector.md

```diff
@@ -0,0 +1,56 @@
+---
+title: '@dbml/connector'
+---
+
+[![NPM](https://img.shields.io/npm/v/@dbml/connector)](https://www.npmjs.com/package/@dbml/connector)
+
+This package is responsible for connecting to a database on your local environment and fetch its schema information.
+
+## Supported Databases
+
+- PostgreSQL
+- MySQL
+- MSSQL
+
+## Prerequisites
+
+Before you begin, ensure you have met the following requirements:
```

Comment:
I think:
We already have **Prerequisites** header. So I think you could shorten this one to:

```markdown
Please ensure you have met the following requirements:
...
```